### ISSUE TITLE
AUC evaluation metrics

### ISSUE THREAD

**Issue by kalifadan** [DFrolova/MULAN#1]
Hey, 
I wonder about your AUC metrics in the HumanPPI task and Metal Binding task.
I saw that SaProt's accuracy on HumanPPI is around 86-87%. Therefore, the AUC of around 0.71 is too low. What am I missing?
I also ran their code and used torchmetrics.AUROC and got higher results.
Thank you!

**Comment by DFrolova** [comment]
Hi!

It is due to the evaluation setup for downstream tasks. We use pure embeddings extracted from prozen PLMs and train a small prediction model on them. In the SaProt paper authors finetune the whole PLM for each downstream task, which should give better results.

Best regards

**Comment by kalifadan** [comment]
Thank you!
